### ISSUE TITLE
[REQUEST] Import from Mapgenie Account

### ISSUE THREAD

**Issue by Coy-Allen** [V1P3R-FMG/free-map-genie#40]
It seems that pre-existing found locations are not transfered when using the extension (chrome). Is it posible to import progress from the logged in account? I found/installed this extension after hitting the 100 location limit.

**Comment by MrFusiion** [comment]
yeah should be possible, Was initially a planned feature in v1 but as I also wanted my progress to be transferred, but while developing this extension lost my progress anyway so never really started working on this feature, But can be useful as many new people probably download this after hitting the locations cap.

**Comment by jojojo8359** [comment]
I know this is probably in the works for a future version, but here's my current band-aid for this (v2.0.6):
- Disable the extension (so you can access your MapGenie account normally) and make sure you're signed in properly
- Open the map you want to transfer data for
- Run this snippet in your browser console (modified from the snippet listed in #48):
```js
window.localStorage.setItem("fmg:game_" + game.id + ":map_" + mapData.map.id + ":user_-1", JSON.stringify({
    "locationIds": Object.keys(user.locations).map(Number),
    "categoryIds": [],
    "notes": [],
    "presets": [],
    "presetOrder": [],
    "visibleCategoriesIds": []
}));
```
(If you're not using "mock user" mode, replace the `":user_-1"` part with `":user_" + user.id`.)

This should transfer your existing locations into the extension format, such that when you re-enable the extension, it should use the locations from your account as a starting point. I highly recommend making a JSON backup after loading it for the first time!

**Comment by MrFusiion** [comment]
Added in [v2.1.0](https://github.com/V1P3R-FMG/free-map-genie/releases/v2.1.0).
